src/promptflow/promptflow/exceptions.py

```diff
@@ -223,19 +225,10 @@ class ValidationException(UserErrorException):
 
 class _ErrorInfo:
     @classmethod
-    def get_error_info(cls, e: Exception):
-        if not isinstance(e, Exception):
-            return None, None, None, None, None
+    def get_error_info(cls, e: BaseException):
```

Comment:
I found the `_ErrorInfo` class is newly added in #1455 and it's purpose is to extract info from both PromptFlowException and other 3-rd party exceptions.

Actually we already have a `ExceptionPresenter` class designed to serve the same purpose:
https://github.com/microsoft/promptflow/blob/ab24a1517f087f2eaaedc3cec443b867c3148ce3/src/promptflow/promptflow/_utils/exception_utils.py#L159

We might consider merge these two classes into one in the future.

Reply:
I have analyzed the codes on both sides and found that they cannot be reused at the code level. The current modifications are all code from the SDK side, so we will not merge them for now. If the codes on both sides can be reused, we will merge them later.